spec/models/post_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
```

Comment:
## Please your test did not pass. Please try again and submit for review 👍 .
- 
![Capture](https://user-images.githubusercontent.com/63972670/188841424-0e1da4fa-93ac-47ec-abc3-f8870f05dc0e.JPG)